LiveVox.NET/Models/Contact/Enumerations/ContactField.cs

```diff
@@ -0,0 +1,467 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace LiveVox.NET.Models.Contact.Enumerations
+{
+    [JsonConverter(typeof(JsonStringEnumConverter))]
+    public enum ContactField
```

Comment:
Consider a better name

Reply:
Renamed it to - SearchConditionContactField